### PR TITLE
fix(components): add my published module page

### PR DESCRIPTION
**Pull request by jennieramida**
## Describe your changes
Fixes: CFE-87, CFE-88

### CODE REVIEW

File: src/lib/components/module/CountBadge.tsx [hunk 1]
```diff
@@ -12,6 +12,6 @@ export const CountBadge = ({ count, variant }: CountBadgeProps) => (
     textColor="text.main"
     p="2px 8px"
   >
-    {count}
+    {count || "0"}
```
Comment (songwongtp) [inline src/lib/components/module/CountBadge.tsx:15]: `{count ?? "N/A"}`

File: src/lib/components/table/modules/ModulesTable.tsx [hunk 2]
```diff
@@ -11,19 +11,23 @@ interface ModulesTableProps {
   modules: Option<ModuleInfo[]>;
   isLoading: boolean;
   emptyState: JSX.Element;
+  isPublishedModules?: boolean;
```
Comment (songwongtp) [inline src/lib/components/table/modules/ModulesTable.tsx:14]: remove `?` and add `isPublishedModules={false}` to the recent module

File: src/lib/layout/navbar/utils.ts [hunk 3]
```diff
@@ -30,6 +30,21 @@ export const getDevSubmenuMove = (isMove: boolean) =>
...
+          submenu: [
+            {
+              name: "My Published Modules",
+              slug: "/my-published-modules",
+              icon: "contract-address" as IconKeys,
+            },
+          ],
+        },
+      ]
+    : [];
```
Comment (songwongtp) [inline src/lib/layout/navbar/utils.ts:47]: move this to before `getWalletSubSectionWasm`

File: src/lib/pages/my-published-modules/components/MyPublishedModulesTable.tsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  }, [keyword, mappedModules]);
+
+  const emptyState = () => {
+    if (error)
+      return (
+        <EmptyState
+          withBorder
+          imageVariant="not-found"
+          message="There is an error during fetching recent modules."
+        />
```
Comment (songwongtp) [inline src/lib/pages/my-published-modules/components/MyPublishedModulesTable.tsx:48]: use `<ErrorFetching message="There is an error during fetching recent modules." />`

File: src/lib/pages/my-published-modules/components/MyPublishedModulesTable.tsx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+      return (
+        <EmptyState
+          imageVariant="empty"
+          message="There is currently no published modules."
+        />
+      );
+    return (
+      <EmptyState
+        withBorder
+        imageVariant="empty"
```
Comment (songwongtp) [inline src/lib/pages/my-published-modules/components/MyPublishedModulesTable.tsx:60]: `imageVariant="not-fount"` ?

**Review by songwongtp — CHANGES_REQUESTED** [review]

**Review by songwongtp — APPROVED** [review]

**Review by evilpeach — APPROVED** [review]

**Pull request merged**